config.go

```diff
@@ -178,6 +178,8 @@ type Config struct {
 
 	// secret for nsqd authentication (requires nsqd 0.2.29+)
 	AuthSecret string `opt:"auth_secret"`
+	// skip using AuthSecret as 'Authorization: Bearer {AuthSecret}' on lookupd queries
+	SkipLookupdAuthorization bool `opt:"skip_lookupd_authorization"`
```

Comment:
Rather than having a double-negative to enable, why not call this `LookupdAuthorization` and default it to `true`?

Reply:
My intention was to structure a setting where the zero value was default; While this struct is always created through `NewConfig` it still seems like good practice to leverage the zero value.

`LookupdAuthorization` is also a good name though, so i don't have a strong objection here. thoughts?